README.md

```diff
@@ -4,45 +4,84 @@
 [![Build Status](https://travis-ci.org/jupyter/notebook.svg?branch=master)](https://travis-ci.org/jupyter/notebook)
 [![Documentation Status](https://readthedocs.org/projects/jupyter-notebook/badge/?version=latest)](http://jupyter-notebook.readthedocs.org/en/latest/?badge=latest)
 
-The Jupyter HTML notebook is a web-based notebook environment for interactive computing.
+The Jupyter notebook is a web-based notebook environment for interactive
+computing.
+
+![Jupyter notebook example](docs/resources/running_code_med.png "Jupyter notebook example")
+
+### Jupyter notebook, the language-agnostic evolution of IPython notebook
+Jupyter notebook is the language-agnostic HTML notebook application for
+Project Jupyter. In 2015, Jupyter notebook was released as part of
+The Big Split™ of the IPython codebase. IPython 3 was the last major monolithic
+release containing both language-agnostic code, such as the *IPython notebook*,
+and language specific code, such as the *IPython kernel for Python*. As
+computing spans many languages, Project Jupyter will continue to develop the
+language-agnostic **Jupyter notebook** in this repo and with the help of the
+community develop language specific kernels which are found in their own
+discrete repos.
+[![The Big Split™ announcement](https://blog.jupyter.org/2015/04/15/the-big-split/)]
+[![Jupyter Ascending blog post](http://blog.jupyter.org/2015/08/12/first-release-of-jupyter/)]
 
-## Usage
+## Installation
+Detailed [Installation documentation](http://jupyter.readthedocs.org/en/latest/install.html)
+can be found on [ReadTheDocs](http://jupyter-notebook.readthedocs.org/en/latest).
```

Comment:
I think it's confusing to have 1 sentence that link to 2 places. Not annoying for this PR in particular, but I would rephase in something like:

```
You can find the installation documentation for the [jupyter platform, on readthedocs](https://jupyter...)
The documentation for advance usage of Jupyter-Notebook can be found [here](https://jupyter-notebook.rtfd...)
```